tapir/coop/tests/test_incoming_payment.py

```diff
@@ -103,3 +104,25 @@ def assert_payment_not_visible(self, response, payment):
             response.content.decode(),
             "Payment should not be visible because it does not concern the logged in member.",
         )
+
+    def test_add_payment_creates_logentry(self):
+        self.assertEqual(CreatePaymentLogEntry.objects.count(), 0)
+
+        self.login_as_user(self.admin_member)
+        response = self.client.post(
+            reverse("coop:incoming_payment_create"),
+            {
+                "paying_member": self.normal_member_1.share_owner.id,
+                "credited_member": self.normal_member_1.share_owner.id,
+                "amount": 100,
+                "payment_date": timezone.now().date(),
+            },
+            follow=True,
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        self.assertEqual(CreatePaymentLogEntry.objects.count(), 1)
+        log_entry = CreatePaymentLogEntry.objects.first()
+        self.assertEqual(log_entry.paying_member.id, self.normal_member_1.id)
+        self.assertEqual(log_entry.amount, 100)
```

Comment:
Nice!
We might as well check that the `credited_member` and the `payment_date` are correct too.

Reply:
`credited_member` has not been part of `CreatePaymentLogEntry` but I've added `payment_date`